.github/workflows/integration-test.yml

```
@@ -193,3 +198,48 @@ jobs:
       - name: Run tests through docker image
         run: |
           docker run pycytominer:testing pytest
+      # Gather and prepare the dynamic version and other vars for use with work below.
+      # For example, 1.1.0.post6.dev0+bc093ef becomes 1.1.0.post6.dev0_bc093ef
+      - name: Gather dynamic version and set env var
+        id: env-vars
+        run: >-
+          echo "CLEAN_VERSION=$(echo ${{ env.sdist_filename }} | tr '+' '_' )" >> "$GITHUB_OUTPUT" &&
+          echo "DATE_TAG=$(date +'%y%m%d')" >> "$GITHUB_OUTPUT"
+      - name: Login to Docker Hub
+        # only run this task if we have a pull request merged or the schedule triggers from main
+        if: github.event.pull_request.merged || ( github.event_name == 'schedule' && github.ref == 'refs/heads/main')
```

I think we want this if statement to avoid triggering off pull requests merged to non-main branches. 
```suggestion
        if: (github.event.pull_request.merged ||  github.event_name == 'schedule') && github.ref == 'refs/heads/main'
```

---

Thanks, great point! Adding this in.